BuildingSync.xsd

```diff
@@ -17787,4 +17790,468 @@
       <xs:enumeration value="Unknown"/>
     </xs:restriction>
   </xs:simpleType>
+  <xs:complexType name="DerivedModelType">
+    <xs:annotation>
+      <xs:documentation>A derived model represents a supervised learning model derived from data presented in a scenario.</xs:documentation>
+    </xs:annotation>
+    <xs:sequence>
+      <xs:element name="DerivedModelName" type="xs:string" minOccurs="0"/>
+      <xs:element name="MeasuredScenarioID" minOccurs="0">
+        <xs:complexType>
+          <xs:attribute name="IDref" type="xs:IDREF" use="required"/>
+        </xs:complexType>
+      </xs:element>
+      <xs:element name="Models">
+        <xs:complexType>
+          <xs:sequence>
+            <xs:element name="Model" minOccurs="0">
+              <xs:complexType>
+                <xs:sequence>
+                  <xs:element name="StartTimestamp" type="xs:dateTime" minOccurs="1" maxOccurs="1"/>
+                  <xs:element name="EndTimestamp" type="xs:dateTime" minOccurs="1" maxOccurs="1"/>
+                  <xs:element name="DerivedModelInputs">
+                    <xs:complexType>
+                      <xs:sequence>
+                        <xs:element name="IntervalFrequency" type="auc:IntervalFrequencyType" minOccurs="0" maxOccurs="1"/>
+                        <xs:element name="ResponseVariable" minOccurs="1" maxOccurs="1">
+                          <xs:complexType>
+                            <xs:sequence>
+                              <xs:element name="ResponseVariableName" type="auc:FuelTypes" minOccurs="1" maxOccurs="1"/>
+                              <xs:element name="ResponseVariableUnits" type="auc:ResourceUnitsType" minOccurs="1" maxOccurs="1"/>
+                              <xs:element name="ResponseVariableEndUse" type="auc:EndUse" minOccurs="1" maxOccurs="1"/>
+                            </xs:sequence>
+                          </xs:complexType>
+                        </xs:element>
+                        <xs:element name="ExplanatoryVariables">
```

Comment:
Are the explanatory variables the `independent variables`? Seems like these could be inputs to the model, right?

Reply:
Yes.  They are still part of the `DerivedModelInputs` element.